src/node_file.cc

```diff
@@ -898,7 +898,7 @@ void Close(const FunctionCallbackInfo<Value>& args) {
     FSReqWrapSync req_wrap_sync;
     FS_SYNC_TRACE_BEGIN(close);
     SyncCall(env, args[2], &req_wrap_sync, "close", uv_fs_close, fd);
-    FS_SYNC_TRACE_END(close);
+    FS_SYNC_TRACE_END(close, "fd", fd);
```

Comment:
How is the fd number helpful in the trace output? My understanding is that, by the time you'll get the trace output, the fds have already been closed, so there's no way to connect the fds to the file paths that were accessed when the process was running.

Reply:
i think we can output some context of the API. If the user knows which file the fd point to, they can know which file the `close` API close  by trace info. But I'm not sure about the specific scenario.

Reply:
> If the user knows which file the fd point to

By logging the raw fds while the program is running?

What use case did you have in mind when you came up with this change? Is there a performance problem you are trying to solve where the fds and file paths are required in the tracing output?

Reply:
> 

I want to collect the data (such as duration, path) of `fs` API.

Reply:
> such as duration

That's understandable because that's the primary outcome of recording the traces but the existing `TRACE_*()` calls already provide that.

> path

How is that helpful in the context of tracing?